app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -70,6 +70,8 @@
 import io.reactivex.rxjava3.subjects.PublishSubject;
 
 public class LocalPlaylistFragment extends BaseLocalListFragment<List<PlaylistStreamEntry>, Void> {
+    public static final long DEFAULT_THUMBNAIL_ID = -1;
```

Comment:
Move this to `PlaylistEntity` and also use it in the `PlaylistStreamDAO` queries